geode-core/src/main/java/org/apache/geode/internal/cache/TXCommitMessage.java

```diff
@@ -1301,14 +1301,22 @@ protected void txApplyEntryOp(FarSideEntryOp entryOp, List<EntryEventImpl> pendi
         /*
          * This happens when we don't have the bucket and are getting adjunct notification
          */
+        long tailKey = entryOp.tailKey;
+        if (entryOp.filterRoutingInfo != null) {
+          if (entryOp.filterRoutingInfo.getMembers()
+              .contains(this.internalRegion.getDistributionManager().getId())) {
```

Comment:
can just use internalRegion.getMyId() instead here.

Reply:
Yes, I changed that. Thanks for the review!